microcosm_flask/tests/test_basic_auth.py

```diff
@@ -32,14 +32,14 @@ def unauthorized():
     response = client.get("/unauthorized")
     assert_that(response.status_code, is_(equal_to(401)))
     data = loads(response.get_data().decode("utf-8"))
-    assert_that(data, is_(equal_to({
-        "code": 401,
-        "message": "The server could not verify that you are authorized to access the URL requested. "
-                   "You either supplied the wrong credentials (e.g. a bad password), or your browser "
-                   "doesn't understand how to supply the credentials required.",
-        "retryable": False,
-        "context": {"errors": []},
-    })))
+    assert_that(data, has_entries(dict(
+        code=401,
+        message="The server could not verify that you are authorized to access the URL requested.  "
```

Comment:
The missing space at the end of the line was making the test fail; and using `equal_to` instead `has_entries` makes it slower to pinpoint the issue.